iidm/iidm-api/src/main/java/com/powsybl/iidm/network/FlowsLimitsHolder.java

```diff
@@ -8,25 +8,30 @@
 
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Optional;
 
 /**
  * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
  */
 public interface FlowsLimitsHolder {
 
     default Collection<OperationalLimits> getOperationalLimits() {
-        return getCurrentLimits() != null ? Collections.singletonList(getCurrentLimits()) : Collections.emptyList();
+        return getCurrentLimits()
```

Comment:
Don't really get why we only return current limits here (but that was already the case before!)

Reply:
It was for backward compatibility!